crates/jstz_cli/src/account.rs

```diff
@@ -86,28 +86,15 @@ fn delete_account(alias: String) -> Result<()> {
         bail_user_error!("The account '{}' does not exist.", alias);
     }
 
-    // Determine the confirmation message based on the login status. Use the name in the message.
-    let confirmation_message = if cfg.accounts.current_alias() == Some(&alias) {
-        format!(
-            "You are currently logged into the account {}. Are you sure you want to delete it? Please type the account name to confirm: ",
-            alias
-        )
-    } else {
-        format!(
-            "Are you sure you want to delete the account {}? Please type the account name to confirm: ",
-            alias
-        )
-    };
-
-    info!("{}", confirmation_message);
-    io::stdout().flush()?;
+    if cfg.accounts.current_alias() == Some(&alias) {
+        warn!("You are currently logged into the account: {}.", alias);
+    }
 
-    let mut input = String::new();
-    io::stdin().read_line(&mut input)?;
+    let confirmation_alias: String = Input::new().with_prompt(format!("Are you sure you want to delete the account {0}? Please type '{0}' to confirm", alias)).interact()?;
```

Comment:
this is open to spelling errors, can it loop with either type the explicit confirmation or `n/no/abort` to abort? (unless of course the account is called `n` `no` or `abort`).

Reply:
True, but there is nothing stopping the user from pressing [Up Arrow] after the command exits to re-run the command

Reply:
Good enough for me 😃